test/e2e/inspect_test.go

```diff
@@ -479,11 +482,14 @@ var _ = Describe("Podman inspect", func() {
 		ulimits := data[0].HostConfig.Ulimits
 		Expect(ulimits).ToNot(BeEmpty())
 		found := false
+		var limit syscall.Rlimit
+		err := syscall.Getrlimit(unix.RLIMIT_CORE, &limit)
+		Expect(err).ToNot(HaveOccurred())
 		for _, ulimit := range ulimits {
 			if ulimit.Name == "RLIMIT_CORE" {
 				found = true
-				Expect(ulimit.Soft).To(BeNumerically("==", -1))
-				Expect(ulimit.Hard).To(BeNumerically("==", -1))
+				Expect(ulimit.Soft).To(BeNumerically("==", limit.Cur))
+				Expect(ulimit.Hard).To(BeNumerically("==", limit.Max))
```

Comment:
This is a major regression, unlimited must mean unlimited. Everything else just breaks user assumption.

Reply:
Unlimited does not mean unlimited in rootful, since everything has a limit.  The reason for this option for users is to allow them to up the current without effecting the max.  Eliminating this ability means that, this functionality is not available to the users.

Reply:
let's discuss at tomorrows standup.